enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/BehandlingStatistikkDto.kt

```diff
@@ -0,0 +1,24 @@
+package no.nav.familie.kontrakter.ef.iverksett
+
+import java.time.ZonedDateTime
+import java.util.UUID
+
+data class BehandlingStatistikkDto(
+        val behandlingId: UUID,
+        val personIdent: String,
+        val saksbehandlerId: String,
+        val saksnummer: String,
+        val hendelseTidspunkt: ZonedDateTime,
+        val hendelse: Hendelse,
+        val behandlingResultat: String,
+        val resultatBegrunnelse: String,
+        val venteAarsak: String
+)
+
+enum class Hendelse {
+    MOTTATT,
+    PÅBEGYNT,
```

Comment:
Trengs det noe "Venter"/"Pause"- og "Gjenopptatt"-hendelse?

Reply:
Inngår "venter" / "pause" i "mottatt" og / eller "påbegynt" ? Eller tenker du at det er flere nyanser ?

Reply:
Fikk vite at dette er ikke noe vi MÅ ha nå. Fjernet venteAarsak fra dto :) Så da trenger vi ikke flere hendelser for nå. Tenker vi kan oppdatere det når vi skal håndtere flere hendelser i framtiden.